Makefile

```diff
@@ -80,7 +96,34 @@ e2e-ec2helper-test:
 	${GO_TEST}/test/e2e/e2e-ec2helper-test/... -v
 	${DELETE_STACK}simple-ec2-e2e-ec2helper-test
 
-test: unit-test e2e-test
+license-test:
+	${MAKEFILE_PATH}/test/license-test/run-license-test.sh
+
+go-report-card-test:
+	${MAKEFILE_PATH}/test/go-report-card-test/run-report-card-test.sh
+
+spellcheck:
+	${MAKEFILE_PATH}/test/readme-test/run-readme-spellcheck
+
+shellcheck:
+	${MAKEFILE_PATH}/test/shellcheck/run-shellcheck
+
+test: unit-test e2e-test license-test go-report-card-test spellcheck shellcheck
 
 fmt:
-	goimports -w ./ && gofmt -s -w ./
\ No newline at end of file
+	goimports -w ./ && gofmt -s -w ./
+
+homebrew-sync-dry-run:
+	${MAKEFILE_PATH}/scripts/sync-to-aws-homebrew-tap -d -b ${BIN} -r ${REPO_FULL_NAME} -p ${SUPPORTED_PLATFORMS} -v ${LATEST_RELEASE_TAG}
+
+homebrew-sync:
+	${MAKEFILE_PATH}/scripts/sync-to-aws-homebrew-tap -b ${BIN} -r ${REPO_FULL_NAME} -p ${SUPPORTED_PLATFORMS}
+
+## requires a github token
+upload-resources-to-github:
+	${MAKEFILE_PATH}/scripts/upload-resources-to-github
```

Comment:
would it be clearer to just pass GITHUB_TOKEN similar to how we provide Docker creds in other repos

Reply:
GITHUB_TOKEN should already be set in the user's shell, and it will be set in github actions. There's no need to pass it since there's no login command like there is with docker login